main/src/ca/uwaterloo/flix/language/phase/extra/CodeHinter.scala

```diff
@@ -346,7 +346,13 @@ object CodeHinter {
     case Type.KindedVar(_, _, _, _, _) => false
     case Type.Cst(TypeConstructor.True, _) => false
     case Type.Cst(TypeConstructor.False, _) => false
-    case _ => true
+    case _ => tpe.typeConstructor match {
```

Comment:
Is this as hacky as it looks? What's going on here?

Reply:
We should probably just compute the size of the formula or something like that.

Having more than two effect variables is pretty rare. It occurs in only two places in the std lib!